ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -51,10 +51,22 @@
 #include <thread>
 
 
+int gcd (int a, int b) {
```

Comment:
Greatest common divisor?
Rename it to clear name or add at least comment

Reply:
Renamed